spotlight/commands/play.py

```diff
@@ -68,9 +68,17 @@ def _get_item_suggestions(self, parameter: str) -> list:
             description = "genre"
             item = ArtistItem
 
+        def by_name(x):
```

Comment:
Can you please rename this function? I can see that it is looking for special characters, if it is referenced elsewhere it will be make sense.